test/rebay_api_web/controllers/session_controller_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule RebayApiWeb.SessionControllerTest do
+  use RebayApiWeb.ConnCase
+  alias RebayApi.Repo
+  alias RebayApi.Accounts.User
+  alias RebayApi.TestHelpers
+  
+  test "redirects user to Google for authentication", %{conn: conn} do
+    conn = get conn, "/auth/google?scope=email%20profile"
+    assert redirected_to(conn, 302)
+  end
+  
+  test "creates user from Google information", %{conn: conn} do
+    users = User |> Repo.all
+    assert Enum.count(users) == 0
+
+    ueberauth_auth = %{
+      credentials: %{token: "fdsnoafhnoofh08h38h"},
+      info: %{
+        image: "some-image-url.foo",
+        email: "travis@foo.fake", 
+        first_name: "Travis", 
+        provider: "google",
+        token: "foo",
+        },
+      provider: :google
+    }
+    conn = conn
+    |> assign(:ueberauth_auth, ueberauth_auth)
+    |> get("/auth/google/callback")
+
+    users = User |> Repo.all
+    [ user | _ ] = users
+    assert Enum.count(users) == 1
+    assert get_resp_header(conn, "location") == ["http://localhost:3000/login/#{user.uuid}"]
```

Comment:
Where is `localhost:3000` defined when running these tests?

Reply:
In an environment variable--I neglected to change the tests to reference it when I refactored to store it as such.